<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>build</class>
 <widget class="QWidget" name="build">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>build</string>
  </property>
  <widget class="QCheckBox" name="buildcheckBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>build</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="runcheckBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>run</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="buildandruncheckBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>build and run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
